requirements/py310.yml

```diff
@@ -15,10 +15,10 @@ dependencies:
   - cf-units >=3.1
   - cftime >=1.5
   - dask-core >=2022.9.0
-  - libnetcdf <4.9
+  - libnetcdf !=4.9.1
   - matplotlib >=3.5
   - netcdf4
-  - numpy >=1.19
```

Comment:
@trexfeathers Whilst you're here could you bump this from `numpy >=1.19` to `numpy >=1.21`, which we're due to due according to the [NEP29 Drop Schedule](https://numpy.org/neps/nep-0029-deprecation_policy.html#drop-schedule)